chart/values.yaml

```diff
@@ -1,8 +1,9 @@
 image:
   empty: rancher/pause:3.1
-  # elemental-operator image is also build on registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-operator
+  # elemental-operator image is also build on:
+  #   registry.opensuse.org/isv/rancher/elemental/teal52/15.3/rancher/elemental-operator
   repository: quay.io/costoolkit/elemental-operator
-  tag: ""
+  tag: 0.0.0-noversion
```

Comment:
cant we set `latest` here? That will point to a valid version in case we screw this up.